### ISSUE TITLE
plugins generally broken

### ISSUE THREAD

**Issue by ghost** [ether/etherpad-lite#1039]
Hi,

I don't know if plugins have proper version control, but plugins should only be shown if they actually work. 
Quick tests show that a considerable number (esp. of the more complex plugins) do not work. I run EPL a2394fc, node 0.6.19 (I know I know it's old)

There are currently 24 plugins listed. In release 1.1.2, the following do not appear to work
- sktechpad (does takes the left half of the screen, but nothing is shown)
  *sketchpad_pdfupload (dependend on image_convert, file_upload, sketchpad)
- file_upload (error: Uncaught ReferenceError: AjaxUpload is not defined in http://IP:9001/static/plugins/ep_fileupload/static/js/fileUpload.js (line 1))
- image_convert (as it depends on file_upload)
- embed_media (button has no graphic, does not appear to be doing anything)
  *tokbox works, but appears to have a global chatroom, not sure that is what is indented
- datatables failes to load (there is a thread somewhere here about a workaround though)
- sociallinks no extra buttons show up

Some that I have tried that work include twitter, linkify, headings. I have not tested all, and I might have issues because of configuration etc.

Bottom Line: EPL is really great, and I am very happy about the continuing work that you guys are doing, but the plugins are frustrating to use as they are clearly not working as they should (in general)
Maybe, a more stringent testing or a requirement to clearly designate the versions with which plugins work would help with this.

PS: there was an earlier thread about the plugins being dramatically broken, feel free to merge...
